base/src/main/java/uk/ac/ebi/atlas/model/experiment/ExperimentType.java

```diff
@@ -80,13 +83,24 @@ public static ExperimentType get(String experimentTypeDescription) {
     }
 
     public static boolean containsBaseline(Set<String> experimentTypes) {
-        //return experimentTypes.contains(RNASEQ_MRNA_BASELINE.getDescription()) || experimentTypes.contains(PROTEOMICS_BASELINE.getDescription());
-        return experimentTypes.contains(RNASEQ_MRNA_BASELINE.name()) || experimentTypes.contains(PROTEOMICS_BASELINE.name());
+        // return experimentTypes.contains(
```

Comment:
Lots of commented out code around here... Probably out of scope for a linting job though.